include/GafferImageUI/ImageView.h

```diff
@@ -108,7 +114,9 @@ class ImageView : public GafferUI::View
 		static void registerDisplayTransform( const std::string &name, DisplayTransformCreator creator );
 		static void registeredDisplayTransforms( std::vector<std::string> &names );
 
-	protected :
+        ImageViewSignal &zoomLevelChangedSignal();
```

Comment:
Given that we have `zoomLevelPlug()`, and `plugSetSignal()`, do we actually need a separate `zoomLevelChangedSignal()`? Gaffer generally pursues a policy of "plugs are the primary public interface to a node", and only adds additional stuff when absolutely necessary.